docs/core/linux-prerequisites.md

```diff
@@ -20,9 +20,13 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 ## Supported Linux versions
 
+# [.NET Core 2.x](#tab/netcore2x)
+
 .NET Core 2.x is supported on the following Linux distributions/versions:
 
- * Red Hat Enterprise Linux 7 x64
+> [!NOTE]
+> Red Hat will be providing a distribution of .NET Core 2.0 very soon. For more information, see [the Microsoft .NET download center](https://www.microsoft.com/net/core#linuxredhat).
```

Comment:
I'd leave the article out of the link. Also, I don't usually refer to that page as a download center. Maybe ".NET Core Downloads - Linux" page?

will be providing -> will provide

very soon -> soon